packetbeat/protos/cassandra/config.go

```diff
@@ -0,0 +1,35 @@
+package cassandra
+
+import (
+	"fmt"
+	"github.com/elastic/beats/packetbeat/config"
+	"github.com/elastic/beats/packetbeat/protos"
+	"github.com/pkg/errors"
+)
+
+type cassandraConfig struct {
+	config.ProtocolCommon `config:",inline"`
+	SendRequestHeader     bool   `config:"send_request_header"`
+	SendResponseHeader    bool   `config:"send_response_header"`
+	Compressor            string `config:"compressor"`
+	OPsIgnored            string `config:"ignored_ops"`
```

Comment:
alternatively you can use the ucfg Unpacker interface to push parsing frame ops into the config loading phase (better error message naming exact faulty setting) like this:

```
type cassandraConfig struct {
    ...
    OpsIgnored []FrameOp
    ...
}
```

with

```
func (f *FrameOp) Unpack(in interface{}) error {
    s, ok := in.(string)
    if !ok {
      return errors.New("expected string")
    }

    op, err := FrameFromOpString(s)
    if err != nil {
        return err
    }

    *f = op
}
```